source/compatibility.tex

```diff
@@ -1151,7 +1151,7 @@
 that interacts with newer \Cpp{} code using regions declared reachable may
 have different runtime behavior.
 
-\diffrefs{refwrap}{arithmetic.operations}, \ref{comparisons},
+\diffref{refwrap,arithmetic.operations,comparisons},
 \ref{logical.operations}, \ref{bitwise.operations}
```

Comment:
There are additional `\ref`s here.

Please also check the other cases, I haven't looked thoroughly.

Reply:
Good catch! I didn't notice any others.